less/layout/_navbar.less

```diff
@@ -213,6 +227,18 @@
     }
 }
 
+// Prevent menu from overflowing on the right side of the screen. The reason we
+// are selecting the 2nd last child is because the last child is a mobile-only
+// menu item. 
+#main-menu .navbar-nav li.dropdown:nth-last-child(2) { 
```

Comment:
It might seem a bit hacky. I wanted to add a custom parameter to the menu item in menu.toml. Something like `class` or `direction`. However, Hugo doesn't allow params on menu items, only menus.

I also tried doing `:last-of-type`, but that selector only works with elements, not classes. That would've been perfect if it worked that way because I would select the last `.dropdown-menu`.